projects/uitdatabank/models/common-contributors.json

```diff
@@ -0,0 +1,17 @@
+{
+  "title": "common-contributors",
+  "type": "array",
+  "minItems": 1,
+  "uniqueItems": true,
+  "description": "An array of emails of users who need to have edit rights.",
+  "items": {
+    "type": "string",
+    "format": "email"
```

Comment:
After [seeing here](https://github.com/cultuurnet/udb3-backend/pull/1253/files#diff-689482cf065520db8fab34b8dc4c31cf5cf0d726e308cadcdb2e365264e31473R18) that the emails can be at most 255 characters long, I think we should add a `"maxLength": "255"` here. (We probably also need to add that in other existing schemas as well, but for new ones I would try to include DB constraints like that already)